explainer-server/app/views/explainEditor.scala.html

```diff
@@ -48,14 +51,23 @@
         function updateWordCountDisplay(){
             $(".top-toolbar__word-count-wrapper__count").text(getBodyWordCount())
         }
-        function updateWordCountWarningDisplay(){
-            if(getBodyWordCount()>maxWordCount){
+        function updateWordCountWarningDisplay() {
+            if (getBodyWordCount() > maxWordCount) {
                 $(".top-toolbar__word-count-wrapper__warning-outer").show();
-            }else{
+            } else {
                 $(".top-toolbar__word-count-wrapper__warning-outer").hide();
             }
         }
 
+        function updateStatusBar(message){
+            if(message.length>0){

```

Comment:
Yes. I guess it's having done Ruby, where anything non `null` or `false` evaluates to `true` :)

I like being explicit, so I would vote to leave it like that (unless you feel strongly about it ^^)